src/main/java/hello/HelloWorldHttpController.java

```diff
@@ -16,6 +16,8 @@ public String greeting(@RequestParam(value="name", required=false, defaultValue=
         if (true){
             int r = 4 + 5;
             System.out.print(r);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AS106)